bundler-parceler/src/main/java/com/hannesdorfmann/fragmentargs/bundler/ParcelerArgsBundler.java

```diff
@@ -15,7 +15,7 @@
     bundle.putParcelable(key, Parcels.wrap(value));
   }
 
-  @Override public <V> V get(String key, Bundle bundle) {
+  @Override public Object get(String key, Bundle bundle) {
```

Comment:
Ah, I assembled test project and I see the problem now. I updated the PR. 

My proposition is to remove inferred return type from interface and add strict casting to generated code. Pros are obvious but there is one caveat. User can specify the inheritor of arg class and try to use the bundler for its super class. In this case we get ClassCastException I think. Although I don't think somebody will do something like that (only mistakenely maybe).